Jenkinsfile

```diff
@@ -239,6 +239,9 @@ pipeline {
                     environment {
                         GIT_CACHE_PATH = "${HOME}/cache"
                         SCCACHE_BUCKET = credentials("brave-browser-sccache-mac-s3-bucket")
+                        SIGN_WIDEVINE_CERT = credentials("widevine_brave_prod_cert.der")
+                        SIGN_WIDEVINE_KEY = credentials("widevine_brave_prod_key.pem")
+                        SIGN_WIDEVINE_PASSPHRASE = credentials("447b2fa7-c989-43af-9047-8ae158fad0a3")
```

Comment:
We'll need to update the build here:
https://github.com/brave/devops/blob/master/jenkins/jobs/brave-browser-build.yml

Do we need the above though? since it's running with `--skip_signing` (this Jenkinsfile isn't used for official builds, only PR builder)

Reply:
@bsclifton  `--skip_signing` is not directly related with vmp.
There are two kinds of signing.
One is for binary itself and the other is signature file generation for vmp.
These variables are for sig file generation.

Reply:
We can also skip sig file generation on PR builder by not setting above variables.
If os env doesn't have them, vmp feature is disabled automatically with this PR.

What I want to check more in PR build is vmp works well regardless of binary signing.
We only checked vmp works with unsigned binary.

Reply:
@simonhong ah - so `--skip_signing` only skips the executable signing (ex: with `codesign` or the `signtool.exe`)

Reply:
> Regardless, I believe we'll need a PR also for the https://github.com/brave/devops/blob/master/jenkins/jobs/brave-browser-build.yml which adds those 3 variables (unless they are already set globally on the instance). This Jenkinsfile is only used for PR builder

@mihaiplesa  Can you check this ?

Reply:
once we make this work we'll add to that Jenkins YML file that's use to describe the build process for public builds